hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncProcess.java

```diff
@@ -436,7 +434,6 @@ public void setOperationTimeout(int operationTimeout) {
    * @return pool if non null, otherwise returns this.pool if non null, otherwise throws
    *         RuntimeException
    */
-  @VisibleForTesting
```

Comment:
There is some kind of compiler problem with this particular annotation and this particular unit. The annotation is available on the classpath and other kinds of references (classes, methods) from the same shaded package are fine.

I would argue that overall our usage of VisibleForTesting brings a binding to Guava for no good reason. We could make our own annotation that communicates the same things. However, I do not propose removal or replacement of VisibleForTesting in this PR (even though it would be a good long term thing), only specifically where it is causing odd compiler problems. The compiler used Java 11+7 GA. There are two files affected in two modules.